library/src/main/java/com/schibsted/spain/barista/interaction/BaristaKeyboardInteractions.kt

```diff
@@ -0,0 +1,24 @@
+package com.schibsted.spain.barista.interaction
+
+import android.support.annotation.IdRes
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.action.ViewActions.closeSoftKeyboard
+import android.support.test.espresso.action.ViewActions.pressImeActionButton
+import android.support.test.espresso.matcher.ViewMatchers.isRoot
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import com.schibsted.spain.barista.interaction.BaristaScrollInteractions.safelyScrollTo
+import com.schibsted.spain.barista.internal.matcher.DisplayedMatchers.displayedAnd
+
+object BaristaKeyboardInteractions {
+
+    @JvmStatic
+    fun close() {
+        onView(isRoot()).perform(closeSoftKeyboard())
+    }
+
+    @JvmStatic
+    fun pressImeActionButton(@IdRes editTextId: Int) {
```

Comment:
An idea, if you want the challenge:
we could try to press on the currently focused EditText, avoiding the need to specify a view id. It would be similar to what we do with `findDrawerMatcher()` in `BaristaDrawerAssertions`, for example, using a `hasFocus() + isAssignableFrom(EditText)` matcher.

Reply:
I will check that code out and modify the function accordingly 👍